main.go

```diff
@@ -11,20 +11,16 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var (
-	version = "dev"
-	commit  = "none"
-	date    = "unknown"
-)
-
 type options struct {
-	Version    bool   `long:"version" description:"Show tool version"`
+	Version    string `long:"version" description:"Show tool version"  default:"0.0.0"`
```

Comment:
The `options` struct is the list of CLI flags. IMO it's wrong to mix the responsibility of this struct with status display.
One-time need of development not worth confusing users when they call `--help`. If you look for dev needs overrides, use the `HD_`-prefixed env variables in appropriate places.

Reply:
so why do we need the version as part of the cli?

Reply:
oh i understand! my bad here